### PR TITLE
Render web UI URLs for jobs created via `openqa-cli schedule`

### PR DESCRIPTION
**Pull request by Martchus**
* This makes it more convenient to view the jobs manually in the web UI.
* Related ticket: https://progress.opensuse.org/issues/125723

### CODE REVIEW

**Review by kraih — APPROVED** [review]

**Review by okurz — CHANGES_REQUESTED** [review]
We have that presentation for clone-job. Could you please check that we don't have duplication?

Also how about using the short form

```
say ' - ' . $host_url->clone->path("t$_") for @$job_ids;
```

**Comment by Martchus** [comment]
I can check wether I can de-duplicate code. Note that I would prefer the long form. It is more clear/intuitive and faster as it avoids one redirection.

File: lib/OpenQA/CLI/schedule.pm [hunk 1]
```diff
@@ -19,6 +19,11 @@ sub _create_jobs ($self, $client, $args, $param_file, $job_ids) {
     my $json = $tx->res->json;
     push @$job_ids, $json->{id} if defined $json->{id} && ref $json->{id} eq '';
     push @$job_ids, @{$json->{ids}} if ref $json->{ids} eq 'ARRAY';
+    if (my $job_count = @$job_ids) {
+        my $host_url = Mojo::URL->new($self->host);
+        say $job_count == 1 ? '1 job has been created:' : "$job_count jobs have been created:";
+        say ' - ' . $host_url->clone->path("tests/$_") for @$job_ids;
```
Comment (kalikiana) [inline lib/OpenQA/CLI/schedule.pm:25]: We may want this to [match the output style used when cloning](https://github.com/os-autoinst/openQA/blob/master/lib/OpenQA/Script/CloneJob.pm#L226)? That's probably what @okurz was referring to.
Currently `print "Created job #$cloned_job_id: $orig_job->{name} -> $base_url/t$cloned_job_id\n";` is how we print newly cloned jobs.

Comment (kalikiana) [reply lib/OpenQA/CLI/schedule.pm:25]: I'd lean towards using the style used here in both cases if we want that since it's less noisy.

Comment (kraih) [reply lib/OpenQA/CLI/schedule.pm:25]: I never liked the clone job output, this is better.

Comment (Martchus) [reply lib/OpenQA/CLI/schedule.pm:25]: But in this context we don't have the name. I wouldn't do an extra request here to print the name. I would also avoid the `/t` route as it is less clear. It is mainly meant for easier typing when entering a URL manually but not useful here.

Comment (kalikiana) [reply lib/OpenQA/CLI/schedule.pm:25]: Yes. We both already said we like yours better. It should just be consistent.

**Review by mimi1vx — APPROVED** [review]

**Pull request merged**